Services/Mail/classes/Mime/Sender/class.ilMailMimeSenderFactory.php

```diff
@@ -7,14 +7,11 @@
  */
 class ilMailMimeSenderFactory
 {
-    /** @var ilSetting */
-    protected $settings;
+    protected ilSetting $settings;
 
     /** @var ilMailMimeSender[] */
-    protected $senders = [];
-
-    /** @var int */
-    protected $anonymousUsrId = 0;
+    protected array $senders = [];
+    protected ?int $anonymousUsrId = 0;
```

Comment:
`$anonymousUsrId` as a class member should never be `null`, so you can use `int` here.